app/views/CannedResponsesListView/Dropdown/index.tsx

```diff
@@ -14,92 +13,85 @@ import { ILivechatDepartment } from '../../../definitions/ILivechatDepartment';
 const ANIMATION_DURATION = 200;
 
 interface IDropdownProps {
-	theme?: TSupportedThemes;
 	currentDepartment: ILivechatDepartment;
 	onClose: () => void;
 	onDepartmentSelected: (value: ILivechatDepartment) => void;
 	departments: ILivechatDepartment[];
 }
 
-class Dropdown extends React.Component<IDropdownProps> {
-	private animatedValue: Animated.Value;
+const Dropdown = ({ currentDepartment, onClose, onDepartmentSelected, departments }: IDropdownProps) => {
+	const animatedValue = useRef(new Animated.Value(0)).current;
+	const { theme } = useTheme();
```

Comment:
`theme` is deprecated on useTheme.
We've been using `colors`, just like https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/fb9c271fa4cdeb47c6c7e53b031c5850f0d3d2d5/app/containers/Chip/index.tsx#L44